projects/storefrontlib/shared/components/media/media.config.ts

```diff
@@ -29,6 +33,33 @@ export abstract class MediaConfig {
     [format: string]: MediaFormatSize;
   };
 
+  /**
+   * Picture element configuration holds the media queries assigned to
+   * a format.
+   * The order of formats matters.
+   * <source> elements in <picture> will be sorted based on this order.
+   * This is necessary because the browser evaluates each
+   * <source> element in order and uses the first one that matches.
+   */
+  pictureElementFormats?: {
+    [format: string]: {
+      mediaQueries: PictureElementQueries;
```

Comment:
`query:` shouldn't this propert be optional as well? I'm thinking about a scenario where the user wants to leave the default mediaQueries for a given format, but in the config would like to additionally pass `width` and `height`

Reply:
Yeah, You have right, changed